README.md

```diff
@@ -128,14 +127,13 @@ fn run() -> Result<(), Error> {
  passing it to `Elasticsearch::new()` create a new instance of `Elasticsearch`
 
 ```rust,no_run
+use url::Url;
```

Comment:
`rust fmt` will order this after `elasticsearch::...`, so should move it to after